### ISSUE TITLE
Can't get you performance in the paper

### ISSUE THREAD

**Issue by unigxy** [Zxy-MLlab/BadVLA#5]
Hello, thanks for your work!

I am attempting to reproduce results on the libero_goal_no_noops dataset using 2x A800 GPUs.

My Stage 1 hyperparameter configuration is as follows:

Learning Rate: 2e-4
Batch Size: 4
Max Steps: 3000
The losses obtained in Stage 1 were:

consistency_loss: 0.019
dissimilarity_loss: -0.85

However, in Stage 2, after training for only 1000 steps, both SR (w/o) and SR (w) reached 90+, indicating that the attack failed completely. Is there something I might have overlooked?

(Note: I also tried running Stage 2 for 15,000 steps, but the SR (w/o) and SR (w) remained very high.)
